seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -1595,9 +1595,9 @@
       "title": "Store Multiple Values in one Variable using JavaScript Arrays",
       "description": [
         "With JavaScript <code>array</code> variables, we can store several pieces of data in one place.",
-        "You start an array declaration with an opening square bracket, end it with a closing square bracket, and put a comma between each entry, like this:<br><code>var sandwich = [\"peanut butter\", \"jelly\", \"bread\"]</code>.",
+        "You start an array declaration with an opening square bracket, end it with a closing square bracket, and put a comma between each entry, like this:\n<br><code>var sandwich = [\"peanut butter\", \"jelly\", \"bread\"]</code>.",
```

Comment:
Could you please move `<code>var sandwich = [\"peanut butter\", \"jelly\", \"bread\"]</code>.` on to it's own line in `basic-javascript.json`.
It would look something like:
```
"You start an array ............... like this: ",
"<code>var sandwich = [\"peanut butter\", \"jelly\", \"bread\"]</code>.",
"<h4>.......</h4>",
```
NOTE: We wont need the `<br>` tag with this mothed.